src/utilities.js

```diff
@@ -47,3 +47,19 @@ export function prettyPrintCamelCase(str) {
     }
     return resultStr;
 }
+
+// Transforms VARROCK_MINE -> Varrock Mine
+export function prettyPrintConstant(str) {
+    let resultStr = "";
+    const spaceStr = str.replace("_", " ");
+    for (let i = 0; i < spaceStr.length; i++) {
+        if (i == 0 || spaceStr.charAt(i-1) == " ") {
+            resultStr += spaceStr[i].toUpperCase();
```

Comment:
Is there a difference btwn using .charAt() or [i]? If not, let's use the same to be consistent

Reply:
I think the only difference is charAt returns "" if you go out of range whereas [] returns undefined. This was just a mistake though

Reply:
Changed here https://github.com/EthanVieira/rs-clicker/pull/48/commits/2508de4ec12ce492b7f36b20ba0644379db853d7

Reply:
There's one more a couple lines under 😄

Reply:
Alright https://github.com/EthanVieira/rs-clicker/pull/48/commits/a59b4519bcb36cfae50a00740189bcc6ee36cd00